packages/core/src/utilities/updateReferencedPlane.ts

```diff
@@ -0,0 +1,54 @@
+import type { Point3, ViewReference } from '../types';
+import { isEqual } from '../utilities/isEqual';
+import { vec3 } from 'gl-matrix';
+
+/** Updates the referencedPlane inside the view reference */
+export function updateReferencedPlane(
+  points: Point3[],
+  reference: ViewReference
+) {
+  if (!points?.length || !reference.FrameOfReferenceUID) {
+    return;
+  }
+  reference.referencedPlane ||= {
+    FrameOfReferenceUID: reference.FrameOfReferenceUID,
+    point: points[0],
+    inPlaneVector1: null,
+    inPlaneVector2: null,
+  };
+  const { referencedPlane } = reference;
+
+  if (points.length === 1) {
+    referencedPlane.inPlaneVector1 = null;
+    referencedPlane.inPlaneVector2 = null;
+  } else {
+    referencedPlane.inPlaneVector1 = <Point3>(
+      vec3.sub(vec3.create(), points[0], points[Math.floor(points.length / 2)])
+    );
+    vec3.normalize(
+      referencedPlane.inPlaneVector1,
+      referencedPlane.inPlaneVector1
+    );
```

Comment:
can you do the other way around. 

v1 = vec3.sub(...)
vec3.normalize(v1,v1)
referenced.inplane1= v1